src/theme/foundations/textStyles.ts

```diff
@@ -10,7 +10,7 @@ export const textStyles = {
   h2: {
     fontFamily: 'heading',
     fontWeight: 400,
-    fontSize: '1.5rem',
+    fontSize: '1.75rem',
```

Comment:
<img width="978" alt="image" src="https://user-images.githubusercontent.com/54227730/204615643-9899ba1c-29c8-4cd4-aee3-661902244606.png">

```suggestion
    fontSize: { base: '1.5rem', md: '1.75rem' },
```

Reply:
thx!